base/applications/taskmgr/graphctl.c

```diff
@@ -251,14 +259,16 @@ void GraphCtrl_InvalidateCtrl(TGraphCtrl* this, BOOL bResize)
     /*   LineTo(m_dcGrid, m_rectPlot.left, m_rectPlot.top); */
 
     /*  draw the horizontal axis */
-    for (i = this->m_rectPlot.top; i < this->m_rectPlot.bottom; i += 12)
+    for (i = this->m_rectPlot.top; i < this->m_rectPlot.bottom; i += this->m_nGridYPixels)
     {
         MoveToEx(this->m_dcGrid, this->m_rectPlot.left, this->m_rectPlot.top + i, NULL);
         LineTo(this->m_dcGrid, this->m_rectPlot.right, this->m_rectPlot.top + i);
     }
 
     /*  draw the vertical axis */
-    for (i = this->m_rectPlot.left; i < this->m_rectPlot.right; i += 12)
+    /*  In order to keep grid position uniform when resizing, vertical axis should be  */
+    /*  drawn from right to left  */
+    for (i = this->m_rectPlot.right - this->m_nGridOffsetPixels; i > this->m_rectPlot.left; i -= this->m_nGridXPixels)
```

Comment:
I tested it, and lines on both edges were properly drawn. I have no idea why it's like this, but I think I would decide not to touch the old logic if it's not broken.

Reply:
You say excluding/including a different edge has no impact on actual drawing: that feels somewhat odd.
Well, my point is to investigate/understand what this code actually does, to document/improve/fix it...

Reply:
But honestly that's way beyond my quick & dirty hack... I don't know whether I have the motivation to discover other parts of Task Manager or GDI APIs for this(mainly because of having little time).

If you'd give me a month or so, I think I will find some time to sort things out.

Reply:
I've dived into it and checked; this function is called when the control is invalidated (noramlly resizing) so I didn't really notice the incorrect painting. Yes, the edge shouldn't be excluded. I'll fix this in next commit.